mp/src/game/shared/momentum/weapon/weapon_shareddefs.h

```diff
@@ -5,6 +5,8 @@
 #define MAX_WEAPON_PREFIX 16
 #define WEAPON_PRINTNAME_MISSING "!!! Missing printname on weapon"
 
+#define MOM_WEAPON_STICKYBOMB_COUNT 3
```

Comment:
Should this be hard coded to 3 when there are plans for the trigger to set a custom limit?

Reply:
For the context of this PR, yes. When we get around to doing the custom trigger we will revisit. This serves as the default limit, and the trigger is going to limit how many can be fired in it as well as how many can be out at a time, but all of that logic will be in the player.